### DISCUSSION TITLE
respect！

### DISCUSSION THREAD

**Discussion by MaoXinn** [WeNeedHome/SummaryOfLoanSuspension#551] [General]
hhhhhhh，简直太好笑了，看来我还是高估repo内的阅读水平了
我居然还能被当成五毛骂，看来真的是忠诚不绝对=绝对不忠诚了
respect！

**Comment by zcaaron** [comment]
所以应该怎样

**Reply by MaoXinn** [reply]
(#^.^#)

**Reply by liuwenwu520** [reply]
应该带头组织这些人进行游行，而不是打嘴炮

**Comment by zxcs001** [comment]
说实话，自由派反贼现在真没几个了。现在反贼已经是少数，更多的人是不关注和反感。

海外华人也就是看个乐子，国内被铁拳锤的越狠，这帮人乐子越多。

你不知道20年之后，海外华人各种捐款送物资的。

今年广东发大水有几个？去年河南水灾有几个？

现在是连关注都不关注了，出了事就是看热闹，等烟花的。

**Reply by MaoXinn** [reply]
(#^.^#)

**Reply by last77** [reply]
墙内出事了邻居都是看热闹 还指望墙外的帮 想啥呢

**Comment by nitamademalegebi** [comment]
所以到底是什么旗帜呢，你把话说清楚

**Comment by chenhaizano** [comment]
你还没说是什么旗啊

**Comment by LTaooo** [comment]
我真的对你们这帮无脑粉红厌恶至极, 真正阻挡社会进步的垃圾

**Reply by MaoXinn** [reply]
扣帽子还是你会扣，(#^.^#)

**Comment by ghost** [comment]
你所谓的自由派反贼，其实是那些一次又一次忍受着自己权力被剥夺，最后忍不住想要发声的人罢了。当铁拳砸到你头上的时候，你跳脚的估计比谁都凶。

**Comment by nitamademalegebi** [comment]
你个垃圾，你指的到底是什么旗帜啊，赶紧他妈的说啊

**Comment by MaoXinn** [comment]
(#^.^#)

**Reply by nitamademalegebi** [reply]
你个垃圾傻逼东西说的鬼东西

**Comment by nitamademalegebi** [comment]
你妈逼没有买到烂尾楼，你就不会说出这种傻逼的话，祝你早日被铁拳暴打

**Reply by 710baichenshan** [reply]
宝贝，别急啊。 你这么急，就成了别人的乐子。 事实上我们屌用没有

**Comment by dongyc519** [comment]
虽然是事实，但是不代表这样既是合理合法，多数人团结也不一定代表正义和真理。

**Comment by JACWATA** [comment]
你也知道动不动被别人扣帽子难受？
